<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Melvin Strobl</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UNote</string>
  </property>
  <property name="toolTipDuration">
   <number>5</number>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1680</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionLoad_Pattern">
   <property name="text">
    <string>Load Pattern</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionOpen_Register_Map">
   <property name="text">
    <string>Open Register Map</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionCOM">
   <property name="text">
    <string>port</string>
   </property>
  </action>
  <action name="actionbaud">
   <property name="text">
    <string>baud</string>
   </property>
  </action>
  <action name="dsfa">
   <property name="text">
    <string>Port</string>
   </property>
  </action>
  <action name="actionport">
   <property name="text">
    <string>port</string>
   </property>
  </action>
  <action name="actionbaud_2">
   <property name="text">
    <string>baud</string>
   </property>
  </action>
  <action name="actionDUT_Test">
   <property name="text">
    <string>DUT Test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
